server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -171,29 +183,50 @@ protected void parseConfiguration(ServerConfiguration conf) {
     poolNameToRegexPattern = new HashMap<>();
     for (Entry<String,String> table : t.tableIdMap().entrySet()) {
       tableIdToTableName.put(table.getValue(), table.getKey());
+      conf.getTableConfiguration(table.getValue()).addObserver(this);
       Map<String,String> customProps = conf.getTableConfiguration(table.getValue()).getAllPropertiesWithPrefix(Property.TABLE_ARBITRARY_PROP_PREFIX);
       if (null != customProps && customProps.size() > 0) {
         for (Entry<String,String> customProp : customProps.entrySet()) {
           if (customProp.getKey().startsWith(HOST_BALANCER_PREFIX)) {
+            if (customProp.getKey().equals(HOST_BALANCER_OOB_CHECK_KEY) || customProp.getKey().equals(HOST_BALANCER_POOL_RECHECK_KEY)
+                || customProp.getKey().equals(HOST_BALANCER_REGEX_USING_IPS_KEY) || customProp.getKey().equals(HOST_BALANCER_REGEX_MAX_MIGRATIONS_KEY)) {
+              continue;
+            }
             String tableName = customProp.getKey().substring(HOST_BALANCER_PREFIX.length());
             String regex = customProp.getValue();
             poolNameToRegexPattern.put(tableName, Pattern.compile(regex));
           }
         }
       }
     }
-    String oobProperty = conf.getConfiguration().get(HOST_BALANCER_OOB_CHECK);
+    String oobProperty = conf.getConfiguration().get(HOST_BALANCER_OOB_CHECK_KEY);
```

Comment:
parsing non-table specific configuration changes based on a specific table configuration observer seems hokey, but it would work of course.......